dockerfiles/Dockerfile.sandbox

```diff
@@ -18,11 +18,17 @@ FROM tiangolo/uwsgi-nginx-flask:python3.10
 # Install dependencies required for Lean 4, pypy3, and other tools
 RUN apt-get update && \
     apt-get install -y curl git net-tools bzip2 build-essential libseccomp-dev && \
-    curl -L https://downloads.python.org/pypy/pypy3.10-v7.3.17-linux64.tar.bz2 -o /tmp/pypy.tar.bz2 && \
+    case "$TARGETARCH" in \
```

Comment:
`Unsupported TARGETARCH ''`

@wasiahmad You'll need to add `ARG TARGETARCH` after the first from. Otherwise this fails.